agent/app/args/flag.go

```diff
@@ -21,9 +21,11 @@ const (
 	acceptInsecureCertUsage  = "Disable SSL certificate verification. We do not recommend setting this option"
 	licenseUsage             = "Print the LICENSE and NOTICE files and exit"
 	blacholeEC2MetadataUsage = "Blackhole the EC2 Metadata requests. Setting this option can cause the ECS Agent to fail to work properly.  We do not recommend setting this option"
+	capabilitiesUsage        = "Print the agent capabilities and exit"
 
 	versionFlagName              = "version"
 	logLevelFlagName             = "loglevel"
+	capabilitiesFlagName         = "capabilities"
```

Comment:
Likewise, I'd consider naming the flag `-environmentCapabilities`, `-hostCapabilities` or something similar that makes that association explicit.

Reply:
Since we also support Linux Capabilities in ECS now, naming this to agentCapabilities or something different can make it less confusing.

Reply:
I would vote for `agentCapabilities`, which, with a few docstring changes, would accurately describe how and what these capabilities are IMO.

Reply:
How do we feel about dashes? (ie: `agent-capabilities`) I personally don't like camelCased flags.

Reply:
Naming is hard. What about `runtime-capabilities`? Its invocation would look like:
```sh
$> ./agent -runtime-capabilites
```
camelCased flags are terrible; I don't know what I was thinking.

Reply:
That name implies unintended meaning as `runtime` is a very overloaded term. In this case it could refer to runtime resolution of capabilities or the _runtime_'s capabilties (ie: docker). I'd favor something that is clearly the Agent's capabilities. 

    ./agent -agent-capabilities 

This one still looks the most accurate to me, ignoring the studder in this case. The help message for the flag itself would clarify how the agent-capabilities are determined and their significance.

Reply:
why not `./agent -capabilities`

Reply:
The issue raised is that the term "capabilities" is grossly overloaded in our technical domain. The flag name `agent-capabilities` makes as much sense and has the added benefit of being scoped both in name and in operation to the agent's idea of a "capability".

Reply:
Makes sense. Might I also suggest  `./agent -ecs-attributes`?

Reply:
If we're going that direction, I'd prefer to use a combination: `ecs-capabilities`. Though the data is represented as attributes these attributes are presented and understood as capabilities (they're prefixed with `com.amazonaws.ecs.capability`). Please correct me if I've misunderstood this data. I'm going to prepare a change with the name `ecs-capabilities` for final submission in the meantime.